src/core/Browser.js

```diff
@@ -13,54 +13,71 @@
  * ```
  */
 
-// @property chrome: Boolean; `true` for the Chrome browser.
-const chrome = userAgentContains('chrome');
+function setupBrowser() {
+	if (typeof window === 'undefined') {
+		return {
+			chrome: true,
+			safari: true,
+			mobile: true,
+			pointer: true,
+			touch: true,
+			touchNative: true,
+			retina: true,
+			mac: true,
+			linux: true
+		};
+	}
+	// @property chrome: Boolean; `true` for the Chrome browser.
+	const chrome = userAgentContains('chrome');
 
-// @property safari: Boolean; `true` for the Safari browser.
-const safari = !chrome && userAgentContains('safari');
+	// @property safari: Boolean; `true` for the Safari browser.
+	const safari = !chrome && userAgentContains('safari');
 
-// @property mobile: Boolean; `true` for all browsers running in a mobile device.
-const mobile = typeof orientation !== 'undefined' || userAgentContains('mobile');
+	// @property mobile: Boolean; `true` for all browsers running in a mobile device.
+	const mobile = typeof orientation !== 'undefined' || userAgentContains('mobile');
 
-// @property pointer: Boolean
-// `true` for all browsers supporting [pointer events](https://msdn.microsoft.com/en-us/library/dn433244%28v=vs.85%29.aspx).
-const pointer = !!window.PointerEvent;
+	// @property pointer: Boolean
+	// `true` for all browsers supporting [pointer events](https://msdn.microsoft.com/en-us/library/dn433244%28v=vs.85%29.aspx).
+	const pointer = !!window.PointerEvent;

```

Comment:
You may use something like `!!(window?.PointerEvent)`. Ditto for `navigator` and the like.